docs/how-to/authentication-plugin-examples.md

```diff
@@ -5,41 +5,41 @@ slug: /plugins/guides/authentication
 pagination_next: getting-started/authentication
 ---
 
-# How To Update Your Authentication and Authorization Setup
+# Handle Authentication and Authorization in your Service
 
 Authentication and authorization are important parts of your Amplication service. Amplication provides flexible options for implementing these security measures.
 
 This guide will walk you through the process of updating your authentication and authorization setup in your Amplication-generated service.
 
-## Creating a Service Without Authentication
+## Adding Authentication to Your Service
 
 If you initially created your service without authentication, you can still add it later. Here are the key steps:
 
 1. Go to your service's Plugins page.
-2. Choose and add an Auth Provider plugin (e.g., JWT, Auth0, etc.) that suits your needs.
-3. Configure the Authentication Entity in your service settings.
+2. Navigate to the "Authentication" category in the left sidebar.
+3. Choose and add an Auth Provider plugin (e.g., JWT, Auth0, etc.) that suits your needs.
+4. Install the "NestJS Auth Module" plugin if it's not already installed. It's a pre-requisite if you're adding Authentication to a Node.js service.
+5. Configure the Authentication Entity in your service settings.
 
 For detailed instructions on this process, refer to the [Authentication Entity documentation](https://docs.amplication.com/user-entity).
 
-## Node.js vs .NET Authentication Options
-
-It's important to note that authentication options differ between Node.js and .NET services in Amplication:
-
-- For Node.js services, you can choose from the various authentication plugins mentioned above.
-- For .NET services, ASP.NET Core Identity is currently the primary authentication option. However, more authentication options for .NET will be added in the future.
+After adding the required plugins, you'll need to:
```

Comment:
If we don't detail here how to do these, I guess we need to add links to the relevant documentation pages- 
how one can configure the auth settings in the plugin options? ("see below" is a valid options as well) 
how roles are defined?
etc.